tests/unit/test_scaffold.py

```diff
@@ -1,88 +1,204 @@
 import pytest
+from flask import Flask
+
 from flask_container_scaffold.app_scaffold import AppScaffold
 
-# Tests with testing config true:
+# Tests for base FLASK_SETTINGS
 
 
-def test_config_valid_settings_file(mock_env_vars):
+def test_no_config_valid_settings_file(mock_instance_folder):
     """
-    GIVEN a test mapping is passed to create_app
-    AND a valid SETTINGS and DEFAULTS_FILE env var is set
-    WHEN we try to create the app
+    GIVEN no config is passed into app_scaffold
+    AND a valid settings.cfg file exists in the specified instance path
+    WHEN we create the app
     THEN it creates an instance of the app
+    AND sets any values in the settings file
+    """
+    scaffold = AppScaffold(instance_path=mock_instance_folder)
+    url = 'http://some.site/cgit/plain/additional-tags?h='
+    assert scaffold.app.config.get('GIT_BASE_URL') == url
+    assert scaffold.app.config.get('CUSTOM_SETTINGS') == 'instance/config.yml'
+    assert scaffold.app.config.get('RANDOM_VAL') == 'farkle'
+
+
+def test_settings_required_no_config_no_settings_file_no_env_setting():
     """
-    app = AppScaffold(config={'TESTING': True}).app
+    GIVEN no config is passed into app_scaffold
+    AND no settings.cfg is in the expected location
+    AND we have not set FLASK_SETTINGS
+    WHEN we create the app
+    THEN it raises a RuntimeError
+    """
+    with pytest.raises(RuntimeError):
+        AppScaffold(settings_required=True)
+
+
+def test_no_config_no_settings_file_no_env_setting():
+    """
+    GIVEN no config is passed into app_scaffold
+    AND no settings.cfg is in the expected location
+    AND we have not set FLASK_SETTINGS
+    AND we have set settings_required=False
```

Comment:
I wonder if this is necessary and/or if it would make sense to add this line with settings_required=False to the test above, since that's where it's actually written into the test.

Reply:
Perhaps this is not needed, since False is the default, but agree saying that the above test fails if you set it to true and have no config file is needed to make it more clear..  Will also work up the additional test or two in this next rev.